lib/modules/manager/pip_requirements/index.ts

```diff
@@ -8,7 +8,7 @@ export { extractPackageFile } from './extract';
 export const language: ProgrammingLanguage = 'python';
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)([\\w-]*)requirements\\.(txt|pip)$'],
+  fileMatch: ['(^|/)([\\w-]*)requirements(|-\\w+)\\.(txt|pip)$'],
```

Comment:
```suggestion
  fileMatch: ['(^|/)[\\w-]*requirements(-\\w+)?\\.(txt|pip)$'],
```
this seems a much better regex